CMakeLists.txt

```
@@ -205,28 +205,33 @@ if(OS2) # no static library on os/2
 endif()
 if(SDLSOUND_BUILD_STATIC)
     add_library(SDL2_sound-static STATIC ${SDLSOUND_ALL_SRCS})
+    add_library(SDL2_sound::SDL2_sound-static ALIAS SDL2_sound-static)
     # Don't rename this on Windows, since DLLs will also produce an import
     #  library named "SDL2_sound.lib" which would conflict; Unix tend to like
     #  the same library name with a different extension for static libs, but
     #  Windows can just have a separate name.
     if(NOT MSVC)
         set_target_properties(SDL2_sound-static PROPERTIES OUTPUT_NAME "SDL2_sound")
     endif()
-    target_include_directories(SDL2_sound-static PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>)
-    if(TARGET SDL2::SDL2)
-        target_include_directories(SDL2_sound-static
-            PUBLIC $<TARGET_PROPERTY:SDL2::SDL2,INTERFACE_INCLUDE_DIRECTORIES>)
-        target_link_libraries(SDL2_sound-static PRIVATE $<BUILD_INTERFACE:SDL2::SDL2> ${OPTIONAL_LIBRARY_LIBS} ${OTHER_LDFLAGS})
+    target_include_directories(SDL2_sound-static PUBLIC
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
+    if(TARGET SDL2::SDL2-static)
+        get_target_property(_sdl2_include_dir SDL2::SDL2-static INTERFACE_INCLUDE_DIRECTORIES)
+        target_include_directories(SDL2_sound-static PUBLIC "$<BUILD_INTERFACE:${_sdl2_include_dir}>")
+        target_link_libraries(SDL2_sound-static PRIVATE $<BUILD_INTERFACE:SDL2::SDL2-static> ${OPTIONAL_LIBRARY_LIBS} ${OTHER_LDFLAGS})
     else()
-        target_include_directories(SDL2_sound-static PUBLIC ${SDL2_INCLUDE_DIRS})
-        target_link_libraries(SDL2_sound-static PRIVATE $<BUILD_INTERFACE:${SDL2_LIBRARIES}> ${OPTIONAL_LIBRARY_LIBS} ${OTHER_LDFLAGS})
+        target_include_directories(SDL2_sound-static PUBLIC $<BUILD_INTERFACE:${SDL2_INCLUDE_DIRS}>)
```

Without these, build system paths were exported in `SDL2_sound-static-targets.cmake`.
```suggestion
        target_include_directories(SDL2_sound-static PUBLIC "$<BUILD_INTERFACE:${SDL2_INCLUDE_DIRS}>")
```

---

Fixed on https://github.com/icculus/SDL_sound/pull/81/commits/c9851e15eea3fcd749541c7fb87ed04d50f5b00e